fvm/evm/stdlib/contract.go

```diff
@@ -178,6 +178,37 @@ var InternalEVMTypeDryRunFunctionType = &sema.FunctionType{
 	ReturnTypeAnnotation: sema.NewTypeAnnotation(sema.AnyStructType),
 }
 
+// InternalEVM.dryCall
```

Comment:
nit: can you move it after call?

Reply:
Updated in https://github.com/onflow/flow-go/pull/6838/commits/d1e1e8e9039802d090a93e2515169dfbd37184aa